.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+name: Dapper CI on PR and Merge
+run-name: Release ${{ github.ref_name }}
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

It seems like `release/v0.16` is the default and active branch in this repo. 
```suggestion
      - release/v0.16
```